packages/adders/paraglide/index.ts

```diff
@@ -10,28 +10,12 @@ import {
 	object,
 	variables,
 	exports,
-	kit
+	kit as kitJs
```

Comment:
This is bad, but works for the moment. This is required since our `workspace` has a `kit` property and this import is also named `kit`